.github/workflows/haddock.yml

```diff
@@ -28,3 +28,4 @@ jobs:
         with:
           folder: dist
           target-folder: ${{ github.ref_name }}
+          # single-commit: true
```

Comment:
Although workflow changes do not take effect until pushed to master, there are cases when it does.
If this PR is approved I'll uncomment this line

Reply:
```suggestion
          -- We publish our haddock, which is non-trivially big. So keeping the whole history
          -- is expensive, and anyway we don't need it.
          # single-commit: true
```